doc/changelog/04-tactics/15015-prodn_tactics.rst

```diff
@@ -0,0 +1,10 @@
+- **Changed:**
+  Documentation in the tactics chapter to give the current correct syntax,
+  consolidate tactic variants for each tactic into a single,
```

Comment:
```suggestion
  consolidated tactic variants for each tactic into a single,
```

Reply:
It's an infinitive, sharing an earlier "to": "**to** give ... [to] consolidate".  One could quibble that "many wording improvements" doesn't start with an infinitive, but IMO the change log can be somewhat more terse.

Reply:
Ah OK, rereading it out loud made it make sense. It was a little tricky to parse.